wrapper/lib/directives.js

```diff
@@ -2041,7 +2041,49 @@ export function Directives (plugin) {
           });
         }
       };
-    }])
+    }]);
+
+    function href($location) {
+      return {
+        restrict: 'A',
+        scope: {
+          relative: '='
+        },
+        link: function (scope, element, attrs) {
+          element.bind('click', e => {
+
+            const platformUrl = $location.protocol() + '://' + $location.host();
+            let url = attrs.href;
+            const withinPlatform = url.indexOf(platformUrl) === 0;
+
+            // open external links in new tab
+            if (url.match(/^https?|www|mailto|\/\//) &&
+              (url.indexOf(platformUrl) === -1 || attrs.target === '_blank')) {
+              e.preventDefault();
+              return window.open(url, attrs.target || '_top');
+            } else if (url.match(/^\/|\?|#/) || withinPlatform || scope.relative) {
+              e.preventDefault();
+              if (withinPlatform) {
+                url = url.replace(platformUrl, '');
+              }
+
+              if (scope.relative) {
+                if (url.indexOf('/' !== 0)) {
+                  url = '/' + url;
+                }
+                url = $location.path() + url
+              }
+
+              $location.url(url);
+
+            }
+
+          });
+        }
+      };
+    }
+    plugin.directive('href', ['$location', href])
+    .directive('ngHref', ['$location', href])
```

Comment:
oh interesting, I didn't realize you could overwrite angular stuff this easily...

Reply:
I think it's not overriding it, just executing both
https://stackoverflow.com/questions/18421732/angularjs-how-to-override-directive-ngclick

Reply:
the other answer on that post seems to indicate you can eliminate the behavior of the angular directive, which is the safe route.
https://stackoverflow.com/questions/18421732/angularjs-how-to-override-directive-ngclick#answer-23209542